docs/getting_started_guide/Initial_Configuration.rst

```diff
@@ -212,26 +234,46 @@ configurations.
 |               |   both the SIMP server and SIMP clients in the 'simp',       |
 |               |   environment, ``simp/hieradata/simp_config_settings.yaml``  |
 +---------------+--------------------------------------------------------------+
-|YUM            | - When the local, SIMP-specific YUM repo exists              |
-|               |   (``/etc/yum.repos.d/simp_filesystem.repo``),               |
+| YUM           | - When the ISO-installed, on-server, SIMP-specific YUM repo  |
```

Comment:
Maybe: When the presence of the simp filesystem repo (/foo/bar/simp_filesystem.repo) indicates an ISO-install